user/deployment/s3.md

```diff
@@ -268,7 +268,20 @@ deploy:
 ```
 {: data-file=".travis.yml"}
 
-### Using S3-compatible Object Storage
+## Set dot_match flag to upload files starting with a period
+
+S3 uploads can be set to upload all files starting with a `.`
+
+```yaml
+deploy:
+  provider: s3
+  ..
```

Comment:
Is this meant to be ellipses? I don't think this is necessary.

Reply:
Let's take them out. :)

Reply:
Looking over the doc I see them used a few times, which I believe was the reason for their original inclusion. I think taking them all out would be beneficial and more consistent with other docs, unless I'm mistaken and there are other places we use them?